snuba/migrations/runner.py

```diff
@@ -47,6 +53,34 @@ def __init__(self) -> None:
             ClickhouseClientSettings.MIGRATE
         )
 
+    def show_all(self) -> List[Tuple[MigrationGroup, List[MigrationStatus]]]:
+        """
+        Returns the list of migrations and their statuses for each group.
+        """
+        migrations: List[Tuple[MigrationGroup, List[MigrationStatus]]] = []
+
+        migration_status = self._get_migration_status()
+
+        def get_status(migration_key: MigrationKey) -> Status:
+            return migration_status.get(migration_key, Status.NOT_STARTED)
```

Comment:
Do you actually need this method ?

Reply:
It could probably be inline. But I did it that way in _get_pending_migrations and wanted to make this the same. (they could also both be inline though)